server/app/crud.py

```diff
@@ -155,5 +155,10 @@ def add_vector(
         return new_vector
 
 
-def get_vectors_for_user(database: Session, user: schemas.User) -> List[model.Vector]:
-    return database.query(model.Vector).filter(model.Vector.users.any(id=user.id)).all()
+def get_level0_for_user(database: Session, user: schemas.User) -> List[model.Vector]:
+    "Query all vectors from the database."
```

Comment:
This docstring doesn't really reflect reality -> "... that a given user has access to."

Reply:
You're right! I've updated the docstring.